src/app/_components/footer.tsx

```diff
@@ -0,0 +1,60 @@
+export default function Footer() {
+    return (
+        <div className="w-screen bg-gray-900 text-center">
+            <div className="flex flex-col sm:flex-row min-h-24 text-center items-center justify-center py-5">
+                <div className="w-1/5 flex flex-col">
+                    <p className="font-bold">Quick Links</p>
+                    <a href="" target="_blank" className="hover:text-white/80 hover:underline-offset-2 m-1 font-light">AP [subject]</a>
```

Comment:
This should be a map of an array; don't repeat so often. And also, this is next js, so please use the `<Link />` component instead of `<a>` tags.